### ISSUE TITLE
Profile name with spaces is not being escaped (or wrapped in quotes by regitlint) (custom or built-in)

### ISSUE THREAD

**Issue by Chris-Thompson-bnsf** [sethreno/ReGitLint#18]
```
Package Id                           Version         Commands
------------------------------------------------------------------
jetbrains.resharper.globaltools      2021.3.2        jb
regitlint                            6.0.6           regitlint
```

I have a powershell script set up in order to call the regitlint command, but when doing so with a profile name specified, the cleanup command will throw an error:

```
VERBOSE: dotnet regitlint -s ".\My.sln" -f staged -p "**/*.cs" --print-command --jb --profile="Built-in: Reformat & Apply Syntax Style" --jb --verbosity="WARN"
JetBrains Cleanup Code 2021.3.2
Running on AMD 64 in 64-bit mode, .NET Core 3.1.19 under Microsoft Windows 10.0.19043
Version: 2021.3.2
dotnet tool run jb cleanupcode ".\My.sln"  --profile=Built-in: Reformat & Apply Syntax Style --verbosity=WARN -dsl=GlobalAll -dsl=GlobalPerProduct -dsl=SolutionPersonal -dsl=ProjectPersonal
JetBrains Cleanup Code 2021.3.2
Running on AMD 64 in 64-bit mode, .NET Core 3.1.19 under Microsoft Windows 10.0.19043
Usage: cleanupcode [options] [solution or project file]
Show help: cleanupcode --help
```

However, when I leave off the profile entirely, I can see that regitlint is escaping the built-in profile that it uses:
```
VERBOSE: dotnet regitlint -s ".\My.sln" -f staged -p "**/*.cs" --print-command  --jb --verbosity="WARN"
JetBrains Cleanup Code 2021.3.2
Running on AMD 64 in 64-bit mode, .NET Core 3.1.19 under Microsoft Windows 10.0.19043
Version: 2021.3.2
dotnet tool run jb cleanupcode ".\My.sln"  --verbosity=WARN --profile="Built-in: Full Cleanup" -dsl=GlobalAll -dsl=GlobalPerProduct -dsl=SolutionPersonal -dsl=ProjectPersonal
JetBrains Cleanup Code 2021.3.2
Running on AMD 64 in 64-bit mode, .NET Core 3.1.19 under Microsoft Windows 10.0.19043
```

I have tried various manners of escaping this in my powershell script, but none seem to work.

I think it would make sense for regitlint to escape this parameter on the C# side, instead of requiring callers to always escape it (since it is pretty expected for the profile names to contain spaces, given that the built-in resharper ones use spaces)

**Comment by Chris-Thompson-bnsf** [comment]
I think a better way to state the ask would be to have Regitlint to wrap the profile value in quotes, rather than requiring the caller to escape things.

**Comment by sethreno** [comment]
Hey Chris, I added the --jb-profile option to handle this in the latest version. Thanks for the feedback.

**Comment by lukewlms** [comment]
@sethreno I haven't gotten this to work - getting this error.

> error: Invalid option 'profile'. Error: Requires value. Name of the code cleanup profile. Full cleanup profile is used if nothing specified.

I'm running this command in 6.0.7:

`dotnet regitlint --jb-profile="KeepNamedParams"`

(Also tried with single quotes and with no quotes around profile name.)

I reverted to the double-specify workaround and that seems to be working ok:

`dotnet regitlint --jb --profile --jb --profile=KeepNamedParams`

**Comment by sethreno** [comment]
@lukewlms Thanks for letting me know. This is fixed for real in version 6.0.8.

**Comment by sethreno** [comment]
@lukewlms Also, if you're finding ReGitLint useful please star it. Thanks!
